planner/optimizer/optimizer-core/src/main/java/eu/seaclouds/platform/planner/optimizer/util/DefaultConstants.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2014 SeaClouds
+ * Contact: SeaClouds
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package eu.seaclouds.platform.planner.optimizer.util;
+
+public class DefaultConstants {
+
+   public static final String DEFAULT_BENCHMARK_INFORMATION_PATH    = "./src/main/resources/benchmark-platforms.yml";
```

Comment:
Thank you @kiuby88 . Suggestions on how to do it? 

Next line reads from the resources. 

I can be wrong, but what I've observed in the lasts days implementing and testing this feature is that: testing the feature with the packaged optimizer looks for it in the resources (and the next line gives just the name), while unit tests that have not packaged the code look for them in the path. 

If you give me any link or suggestion on how to implement it, I'm happy to make the modification.